index.md

```diff
@@ -1,15 +1,55 @@
+
+The Data Commons [Knowledge Graph browser](https://datacommons.org/browser/) allows you to peek into the structure of the graph, and the APIs allow you to directly query the parts of the graph (e.g. nodes, triples, etc.).
+
+Importantly, numeric time series data are first-class entities, with "(statistical) variable" being an entity that represents a metric definition, and "observation" being an entity that represents the value of a variable at a specific time. The [Statistical Variable Explorer](https://datacommons.org/tools/statvar) allows you to browse existing variables, and the [Visualization tools](https://datacommons.org/tools/visualization) provide aggregated views of this data over time, geography, or 2-dimensional space. The APIs also allow you to directly query observations. 
+
+The knowledge graph is also mapped to relational tables that allow for [SQL querying](https://docs.datacommons.org/bigquery/) (requiring a [Google Cloud BigQuery](https://cloud.google.com/bigquery) account) and easier joining to other datasets outside of Data Commons. 
+
+<!--To learn more about the data model and key concepts, see [Key concepts](). -->
+
+## An open-source project and website platform
+
+Data Commons is intended to be a community-based resource, where individuals and organizations can contribute data, code, documentation and educational materials. Source code, schemas, and documentation are publicly available at [https://github.com/datacommonsorg](https://github.com/datacommonsorg). 
+
+Google has partnerships with the [United Nations](https://unstats.un.org/UNSDWebsite/undatacommons/sdgs), the [World Health Organization](https://unstats.un.org/UNSDWebsite/undatacommons/areas/1471028664), [One.org](https://datacommons.one.org/), [TechSoup](https://publicdata.techsoup.org/), and many other non-profit, academic, and governmental organizations across the world. We are always looking to expand the available data sources and welcome contributions from data owners around the world.
+
+In addition, Data Commons makes its data and visualizations accessible to any website through [REST](/api/rest/v2/index.html) and [Javascript](/api/web_components/index.html) APIs. 
+
+Finally, Data Commons provides an open-source, [customizable website](/custom_dc/index.html) implementation, for organizations that want to host their own version of a Data Commons website, using their own data and user interfaces. 
```

Comment:
is "customizable website" the best way to describe custom dc? could we call it `[customizable implementation]`

Reply:
Sure.